src/style.css

```diff
@@ -1 +1,34 @@
-/* add styles here */
+/* stylelint-disable no-duplicate-selectors */
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable rule-empty-line-before */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5),
+.stars--6 .stars__star:nth-child(-n + 6) {
```

Comment:
The `.stars--6` selector is defined here, but there is no corresponding HTML element in the `index.html` file with the class `stars--6`. If you do not plan to add a sixth star rating, you might want to remove this selector to keep the CSS clean and relevant.